lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - any argument
+// Result: argument x
+const id = x => x;
+
+// Async identity function
+//   x - any argument
+//   callback - function, (err, data)
+const asyncId = (x, callback) => {
+  callback(null, x);
+};
+
+// Mapable object iteration identity function
+//   items - array
+//   fn - function
+//   callback - function, (err, data)
+const mapIterationId = (items, fn, callback) => {
+  callback(null, items.map(id));
```

Comment:
In this case, `Array.prototype.map()` can be replaced with `Array.prototype.slice()`, this won't modify the behavior:
```suggestion
  callback(null, items.slice());
```

Reply:
Considering this, I think there can be more than just arrays. Perhaps any objects with the map method: https://github.com/metarhia/common/pull/217#discussion_r232493540